tests/vmi_lifecycle_test.go

```diff
@@ -662,8 +663,13 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 			BeforeEach(func() {
 
+				withoutGuestLogs := func(vmi *v1.VirtualMachineInstance) {
```

Comment:
Why don't you add this as a libvmi exported function?

Reply:
Done